app/controllers/ApplicationController.scala

```diff
@@ -17,52 +17,60 @@
 package controllers
```

Comment:
A lot of this is formatting change very difficult to see where the actual code change is.